test/test_cuda.py

```diff
@@ -4996,6 +4996,101 @@ def test_no_triton_on_import(self):
         self.assertEqual(rc, "False", "Triton was imported when importing torch!")
 
 
+class TestMemPool(TestCase):
+    def test_mempool_id(self):
+        pool1 = torch.cuda.graph_pool_handle()
+        pool2 = torch.cuda.MemPool().id
+
+        # first value of id in a user created pool is always zero
+        self.assertEqual(pool1[0] == 0, pool2[0] == 0)
```

Comment:
This doesn't seem like a great test if, e.g., the tests get executed in a different order

Reply:
Agreed. I have changed the `assertGreater(pool2[1], pool1[1])` test to: `self.assertTrue(abs(pool2[1] - pool1[1]) > 0)`. If the tests get executed in any order, we should still see a difference between pool 1 and pool 2.